anvil/config.py

```diff
@@ -201,3 +202,9 @@ def parse_window(self, window: float):
             raise ConfigError("window must be positive")
         log.warning(f"Using user specified window 'S': {window}")
         return window
+
+    def produce_use_multiprocessing(self):
+        """Don't use mp on MacOS"""
+        if platform.system() == "Darwin":
+            return False
+        return True
```

Comment:
That's a shame. Can try to fix later on I suppose.

Reply:
It';s related to that article I sent you a while ago, the short answer is if we can't pickle it, we can't support it on Mac

Reply:
wrt mp that is

Reply:
Anyway the hack is perfectly acceptable to me right now. You can run the code on both platforms but obviously it will be faster on linux

Reply:
I remember that I decided not to use the "standard" multiprocessing.Pool because it requires pickling and at that time this function needing be multiprocessed was an instance method which was complicated to pickle. Now that the correlation function is just a function it might work, but I guess it's not a priority.

Reply:
Nah leave it for now. I'm satisfied I can train and sample with the examples